### PR TITLE
[hi] Localize version_control.md

### PR DESCRIPTION
**Pull request by bishal7679**
Signed-off-by: Bishal Das <bishalhnj127@gmail.com>
Localizes `content/en/versioncontrol.md` to Hindi `content/hi/version_control.md`
Fixes issue :- #1036

### CODE REVIEW

**Comment by bishal7679** [comment]
@Garima-Negi @anubha-v-ardhan @jayesh-srivastava @sayantani11 
Suggest me any changes in my PR. It will be very helpful

File: content/hi/version_control.md
Comment (ShivamTyagi12345) [inline content/hi/version_control.md:9]: यह एक ऐसा

File: content/hi/version_control.md
Comment (ShivamTyagi12345) [inline content/hi/version_control.md:13]: बैकअप

File: content/hi/version_control.md
Comment (ShivamTyagi12345) [inline content/hi/version_control.md:17]: टकराव to भेद

**Review by ShivamTyagi12345 — CHANGES_REQUESTED** [review]

File: content/hi/version_control.md
Comment (bishal7679) [reply content/hi/version_control.md:17]: But @Garima-Negi was suggested to use टकराव

Comment (ShivamTyagi12345) [reply content/hi/version_control.md:17]: In that case we can opt it out

File: content/hi/version_control.md
Comment (jayesh-srivastava) [inline content/hi/version_control.md:2]: Kindly remove the english part.

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]
In the whole file we have to use `करता` and `ऐसा` instead of their counterparts.

File: content/hi/version_control.md
Comment (bishal7679) [reply content/hi/version_control.md:9]: Ok thanks for the suggestion @jayesh-srivastava & @ShivamTyagi12345 
I am changing 
and can you suggest all the changes at one time then it would be better for me

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: content/hi/version_control.md
Comment (bishal7679) [reply content/hi/version_control.md:2]: > Kindly remove the english part.
You mean (Version Control) should be removed?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Not reusable

**Comment by bishal7679** [comment]
@Garima-Negi @jayesh-srivastava @anubha-v-ardhan 
Could you please proceed for merging this PR with the latest suggested commit?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: content/hi/version_control.md
Comment (Garima-Negi) [inline content/hi/version_control.md:13]: @bishal7679 please go through the content again and localize words like ओवरलैपिंग, इम्पोर्टेन्ट etc. I'm sure you'll find  words in Hindi language for these and more such occurrences.

File: content/hi/version_control.md
Comment (Garima-Negi) [inline content/hi/version_control.md:17]: similar to above comments, words like efficiency, modern need to be localized. Otherwise it is only a transliteration, not localization.

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

**Comment by bishal7679** [comment]
@Garima-Negi I have localized all the suggested changes

**Review by sayantani11 — APPROVED** [review]
LGTM!

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Comment by bishal7679** [comment]
@anubha-v-ardhan updated the suggested changes
But in other hindi terms there have been used तकनीकी

**Comment by anubha-v-ardhan** [comment]
@bishal7679, Have we used तकनीकी for the `category: technology` label? Can you point out the term if any, We need to maintain strict standards for the labels and decided on 
```
Technology ->  प्रौद्योगिकी
Property -> गुण
Concept -> अवधारणा
```
as mentioned on our [reference spreadsheet](https://docs.google.com/spreadsheets/d/1MkZXrXre7dFVFmrRIHgHhNOYFEYcJkRaIWc3Dp8M-TQ/edit#gid=0)

**Comment by bishal7679** [comment]
@anubha-v-ardhan all suggested changes have been resolved.
Can you go for merge down it?

File: content/hi/version_control.md
Comment (anubha-v-ardhan) [inline content/hi/version_control.md:2]:
```suggestion
title: संस्करण नियंत्रण (Version Control)
```

Comment (anubha-v-ardhan) [reply content/hi/version_control.md:2]: Sorry, missed this in earlier reviews, we always prefer to add English in brackets for the title.

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Thank you @bishal7679. 
In addition to this, if you can also change the file name from `version_control.md` to `version-control.md` that would be great as we'd prefer to keep the file name same as English.

**Comment by bishal7679** [comment]
@anubha-v-ardhan I am changing the file name

**Comment by bishal7679** [comment]
@anubha-v-ardhan Now all are good to go

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM!

**Comment by anubha-v-ardhan** [comment]
@bishal7679 Please wait for approval from @jayesh-srivastava and @Garima-Negi. We can't merge a PR with a change request ongoing. Thank you

**Comment by Garima-Negi** [comment]
I think a pending change request is from @ShivamTyagi12345  ?

**Comment by ShivamTyagi12345** [comment]
> I think a pending change request is from @ShivamTyagi12345 ?
LGTM

**Comment by ShivamTyagi12345** [comment]
LGTM

**Comment by bishal7679** [comment]
@Garima-Negi Now can you proceed?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Review by Garima-Negi — APPROVED** [review]

**Review by jayesh-srivastava — APPROVED** [review]
Lgtm! Thanks.

**Pull request merged**
